bundles/org.eclipse.swt/Eclipse SWT/gtk/org/eclipse/swt/widgets/Shell.java

```diff
@@ -2029,8 +2029,8 @@ long notifyState (long object, long arg0) {
  */
 public void open () {
 	checkWidget ();
-	bringToTop (false);
 	setVisible (true);
+	bringToTop (false);
```

Comment:
Since many years long this call did **literally nothing**, would be safer to simply remove that?

Reply:
> Since many years long this call did **literally nothing**, would be safer to simply remove that?

@SyntevoAlex : WDYT?